Artesian/Artesian.SDK/Factory/AuctionTimeSerie.cs

```diff
@@ -188,11 +192,12 @@ private async Task _save(
         /// <param name="deferDataGeneration">DeferDataGeneration</param>
         /// <param name="ctk">The Cancellation Token</param> 
         /// <returns></returns>
-        public async Task Delete(LocalDateTime? rangeStart = null, LocalDateTime? rangeEnd = null, string timezone = null, bool deferCommandExecution = false, bool deferDataGeneration = true, CancellationToken ctk = default)
+        public async Task Delete(LocalDateTime? rangeStart = null, LocalDateTime? rangeEnd = null, string? timezone = null, bool deferCommandExecution = false, bool deferDataGeneration = true, CancellationToken ctk = default)
         {
-            var data = new DeleteCurveData(_identifier)
+            var data = new DeleteCurveData
             {
-                Timezone = timezone,
+                ID = _identifier,
+                Timezone = timezone ?? "CET",
```

Comment:
remove this null coalescing. update the DeleteCurveData to have Timezone nullable and not required